src/pages/LogIn/LogIn.module.scss

```diff
@@ -0,0 +1,115 @@
+.inputIsNotCorrect {
```

Comment:
a lot of style duplicates. Create one general style and call it like input. And then add aditional styles for that main className